### PR TITLE
Updated the ages for Magni, Muradin, and Brann to more lore-friendly …

### PR DESCRIPTION
**Pull request by PaperPlate-Github**
…ones.

I made it so there were roughly 20-30 years between brother since that seemed reasonable.

## Changelog:
- Changed Muradin's age to a lore-specific one, and gave Brann and Magni reasonable ages based on Muradin's.

### CODE REVIEW

**Comment by PaperPlate-Github** [comment]
Addresses #145

File: history/characters/17000_28000_bronzebeard.txt [hunk 1]
```diff
@@ -307,7 +307,7 @@
 	trait=education_learning_4 trait=brave trait=patient trait=diligent
 	father=17042	#Horhunn
 	disallow_random_traits = yes
-	463.1.1={ birth=yes trait=creature_dwarf immortal_age=18 }
+	413.1.1={ birth=yes trait=creature_dwarf immortal_age=18 }
 	481.1.1={
```
Comment (ercarp) [inline history/characters/17000_28000_bronzebeard.txt:311]:
```suggestion
	431.1.1={
```

File: history/characters/17000_28000_bronzebeard.txt [hunk 2]
```diff
@@ -38,7 +38,7 @@
 	trait=education_martial_4 trait=trusting trait=brave trait=honest trait=compassionate
 	father=17042	#Horhunn
 	disallow_random_traits = yes
-	451.4.16={ birth=yes trait=creature_dwarf immortal_age=18 }
+	382.4.16={ birth=yes trait=creature_dwarf immortal_age=18 }
 	469.4.16={
```
Comment (ercarp) [inline history/characters/17000_28000_bronzebeard.txt:42]:
```suggestion
	400.4.16={
```

File: history/characters/17000_28000_bronzebeard.txt [hunk 3]
```diff
@@ -24,7 +24,7 @@
 	trait=education_martial_4 trait=patient trait=generous trait=compassionate trait=honest
 	father=17042	#Horhunn
 	disallow_random_traits = yes
-	426.4.16={ birth=yes trait=creature_dwarf immortal_age=18 }
+	365.4.16={ birth=yes trait=creature_dwarf immortal_age=18 }
 	444.4.16={
```
Comment (ercarp) [inline history/characters/17000_28000_bronzebeard.txt:28]:
```suggestion
	383.4.16={
```

**Review by ercarp — CHANGES_REQUESTED** [review]
Great change, but shouldn't the date for which they receive their class trait be also updated to reflect the new birth year? I added suggestions which change these dates so they receive their class traits at age 18, which is what it was before.

File: history/characters/17000_28000_bronzebeard.txt (hunk 3 above)
Comment (PaperPlate-Github) [reply history/characters/17000_28000_bronzebeard.txt:28]: Ok. I wasn't sure if those needed an update or not so I didn't change them.

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**